R/fct_cross.R

```diff
@@ -24,7 +24,8 @@ fct_cross <- function(.f, ..., sep = ":", keep_empty = FALSE) {
     return(.f)
   }
 
-  .data <- lapply(tibble::tibble(.f, !!!flist), check_factor)
+  .data <- suppressMessages(tibble::tibble(.f, !!!flist, .name_repair = "universal"))
```

Comment:
Hmmmmm, why does it get put into a tibble in the first place? I don't think the names are ever actually used?

Reply:
I'm not totally sure, but I think it's just a convenience for getting all the factors together when the length of `flist` is unknown.

I actually think it makes the most sense to just not have `.f` as a separate input...

Reply:
It was actually needed for recycling, so I've bought it back with a test.